lib/src/pages/homepage.dart

```diff
@@ -76,14 +84,32 @@ class _MyHomePageState extends State<MyHomePage> {
               },
             ),
           ),
+          ValueListenableBuilder<DashboardType>(
+            valueListenable: widget.typeSwitch,
+            builder: (BuildContext context, DashboardType value, _) {
+              return DropdownButton<DashboardType>(
```

Comment:
Using this looks good; one other UI element to consider is a TabBar + Tabs. They work really well w/ and Scaffold and AppBar - the tabs have an integrated look w/ the AppBar. This is a good example of using them: https://docs.flutter.dev/cookbook/design/tabs.

Reply:
I tried that, can't get it to work somehow. Maybe in a follow-up PR.